Scripts/accountData.js

```diff
@@ -0,0 +1,25 @@
+const mailFormat = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
+const passwordFormat = /^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,}$/;
+
+// 가입된 회원
+const registeredAccounts = [
+   {email : 'test@codeit.com' , password : 'codeit101'}
+];
+
+
+// 형식 체크 함수
+const formatCheck = function (text) {return text.match(mailFormat);};
+const passFormatCheck = function (password) {return password.match(passwordFormat);};
```

Comment:
이런 경우 화살표 함수를 쓰시면 더 가독성이 좋을 것 같습니다.
함수 표현식으로 쓰시고 싶으실 경우 줄바꿈을 해주시는게 더 가독성에 좋을 것 같습니다~

```suggestion
const formatCheck = (text) => text.match(mailFormat)
const passFormatCheck = (password) => password.match(passwordFormat)
```

Reply:
```suggestion
const formatCheck = function (text) {
  return text.match(mailFormat);
};
const passFormatCheck = function (password) {
  return password.match(passwordFormat);
};
```